### ISSUE TITLE
html-accessibility-Aria-labels

### ISSUE THREAD

**Issue by Amir-Rastkhadiv** [Amir-Rastkhadiv/fun-gym#2]
Aria-labels should be included on your social links (as they don’t have any text content – just an image).
